README.md

```diff
@@ -32,6 +32,18 @@ import '@adyen/adyen-web/dist/adyen.css';
 
 Adyen Web can also be imported with a `<script>` tag, refer to [the documentation](https://docs.adyen.com/checkout/components-web#step-2-add-components) for instructions.
 
+####Please note: 
```

Comment:
There's a missing space. Some editors won't show the content correctly because of that.
```suggestion
#### Please note: 
```